mint-005.md

```diff
@@ -0,0 +1,165 @@
+# mint-005
+
+## 3 Key Joint Custody
+
+## Motivation
+
+To provide a custody arrangement in which an owner of Bitcoin (principle) is able to secure bitcoin by working with an Agent. Unlike existing collaborative custody models up to this point, **where bitcoin keys WITHIN a multisig threshold are shared between principles and agents**, a joint custody model by default requires a threshold of keys by both the Agent & Principle for movement of funds.
+
+This introduces a concept of "Negative Control" where, by default, funds are not able to be moved unless both the Principle and Agent sign the transaction.
+
+In the event the principle loses access to all of their keys, a secondary agent is available to work with the primary agent such that funds can be recovered after a set period of time.
+
+In the unlikely event the Primary agent has lost 2 of their 3 keys, a timelock enabled threshold allows  only 1 of 3 keys from the primary agent to be signed with the secondary agent.
+
+Finally, in the event the principle no longer wishes to work with the agent, say after a contract expires, the custody defers to a set of recovery keys, which can be held either by the principle, or their own delegated managers of the recovery keys. As a result, when enough time has passed, the Principle is able to move Bitcoin unilaterally without having the Agent sign key material.
+
+### More on Timelock Values
+
+There are three timelocks used for this MinT:
+
+1. `smallest_epoch_timestamp` - The smallest epoch timestamp timelock enables a "Asset Recovery" period such that only one of the principle keys is required to sign.
+
+2. `between_epoch_timestamp` - The epoch timestamp value in between the smallest and largest epoch timestamp enables a "Emergency Recover Path". In the event the Principle has lost all of their keys, the Primary agent and Secondary Agent can work together to recover the Bitcoin in the Joint Custody vault.
```

Comment:
In 7bebe7efd503756903ccb9efc65e1e681a5ebb4e:

I think "Emergency Recovery Path" reads better than "Emergency Recover Path".

Reply:
Updated